src/styles/toolbar.styl

```diff
@@ -7,10 +7,15 @@
     .coz-btn:last-child
         margin-right 0
 
+    .fil-toolbar-more-btn
+        @extends $button--secondary
```

Comment:
Can you use `@extend` instead of `@extends`, please? I tend to unify things through every app